tests/src/FunctionalJavascript/CustomFieldSubmissionTest.php

```diff
@@ -192,31 +261,26 @@ public function testSubmitWebform() {
     $this->assertSession()->assertWaitOnAjaxRequest();
     $this->htmlOutput();
 
-    $this->getSession()->getPage()->checkField("civicrm_1_contact_1_cg1_custom_1");
-    $this->getSession()->getPage()->checkField("civicrm_1_contact_1_cg1_custom_2");
-    $this->getSession()->getPage()->checkField("civicrm_1_contact_1_cg1_custom_2_timepart");
-    $this->getSession()->getPage()->checkField("civicrm_1_contact_1_cg1_custom_3");
-    $this->getSession()->getPage()->checkField("civicrm_1_contact_1_cg1_custom_4");
-    $this->getSession()->getPage()->checkField("civicrm_1_contact_1_cg1_custom_5");
-
-    $this->assertSession()->checkboxChecked("civicrm_1_contact_1_cg1_custom_1");
-    $this->assertSession()->checkboxChecked("civicrm_1_contact_1_cg1_custom_2");
-    $this->assertSession()->checkboxChecked("civicrm_1_contact_1_cg1_custom_2_timepart");
-    $this->assertSession()->checkboxChecked("civicrm_1_contact_1_cg1_custom_3");
-    $this->assertSession()->checkboxChecked("civicrm_1_contact_1_cg1_custom_4");
-    $this->assertSession()->checkboxChecked("civicrm_1_contact_1_cg1_custom_5");
-
+    //Enable custom fields.
+    foreach ($this->_customFields as $name => $id) {
+      $this->getSession()->getPage()->checkField("civicrm_1_contact_1_cg1_custom_{$id}");
+      $this->assertSession()->checkboxChecked("civicrm_1_contact_1_cg1_custom_{$id}");
+      if ($name == 'DateTime') {
+        $this->getSession()->getPage()->checkField("civicrm_1_contact_1_cg1_custom_{$id}_timepart");
+        $this->assertSession()->checkboxChecked("civicrm_1_contact_1_cg1_custom_{$id}_timepart");
+      }
+    }
     $this->saveCiviCRMSettings();
```

Comment:
Awesome - we were getting too many custom fields - which in itself was a good thing 🎉👍